src/containerConfig.ts

```diff
@@ -24,7 +24,8 @@ export const registerExternalValues = (options?: RegisterOptions): DependencyCon
   const jobManagerBaseUrl = config.get<string>('jobManager.url');
   const logger = jsLogger({ ...loggerConfig, prettyPrint: loggerConfig.prettyPrint, hooks: { logMethod } });
 
-  const metrics = new Metrics(SERVICE_NAME);
+  const attributes: Attributes = { name: SERVICE_NAME };
+  const metrics = new Metrics(attributes);
```

Comment:
In boilerplate, the Metrics class does not require attributes
```suggestion
  const metrics = new Metrics();
```